worker/scraper.py

```diff
@@ -99,15 +99,21 @@ def _store_items(feed):
             feed.image_url = fp.feed.image.url
             db.session.commit()
         except:
-            img_tags = BeautifulSoup(requests.get(fp.feed.link).content).find_all('img')
-            for image_tag in img_tags:
-                if "logo" in str(image_tag):
-                    feed.image_url = image_tag.get('src') or image_tag.get('href')
-                    db.session.commit()
-                    break
+            try:
+                img_tags = BeautifulSoup(requests.get(fp.feed.link).content).find_all('img')
+                for image_tag in img_tags:
+                    if "logo" in str(image_tag):
+                        feed.image_url = image_tag.get('src') or image_tag.get('href')
+                        db.session.commit()
+                        break
+            except:
+                pass # eventually, default image
```

Comment:
Added this try block b/c some RSS feeds apparently don't include a ```<link>``` entry in their XML. The comment is because we actually do want default images there eventually.